knowledge_base/Environment-Variables.md

```diff
@@ -83,6 +87,10 @@ If you add a new environment variable, you must add documentation here. Please d
 - [WITH_PRERENDER](#with_prerender)
 - [ZAPIER_WEBHOOK_URL_DEV](#zapier_webhook_url_dev)
 
+## ALLOWED_EVENTS
+
+To enable contests, must be set to `ChainEventCreated,ThreadCreated,ThreadUpvoted,ContestContentAdded,ContestContentUpvoted,ContestStarted,OneOffContestManagerDeployed,RecurringContestManagerDeployed`.
```

Comment:
In this case, it's not so much about "setting" to that value, but appending those events to the current value. There are other events not listed there that are required for other features.

Reply:
Will probably switch this back to DISALLOWED events soon since the number of allowed events now exceeds the number of disallowed events (and this will remain true for the foreseeable future).